jung-samples/src/main/java/edu/uci/ics/jung/samples/util/DemoTreeSupplier.java

```diff
@@ -0,0 +1,162 @@
+package edu.uci.ics.jung.samples.util;
+
+import com.google.common.graph.MutableNetwork;
+import com.google.common.graph.NetworkBuilder;
+import edu.uci.ics.jung.graph.CTreeNetwork;
+import edu.uci.ics.jung.graph.MutableCTreeNetwork;
+import edu.uci.ics.jung.graph.TreeNetworkBuilder;
+
+/** @author Tom Nelson */
+public class DemoTreeSupplier {
```

Comment:
There may be some reason why you didn't put these methods in TestGraphs, but if so, I don't understand what it is.

Reply:
I'm not sure what the ultimate fate of TestGraphs will be, and I intended these as helpers to the sample code, not as standards to use or follow while making trees.